flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
More archs are coming (in the next release, in fact)

Reply:
(flathub does have a beta branch where you could put beta manifests prior to their stable release as well)